go/enclave/components/rollup_consumer.go

```diff
@@ -47,15 +48,21 @@ func NewRollupConsumer(
 	}
 }
 
-func (rc *rollupConsumerImpl) ProcessRollupsInBlock(ctx context.Context, b *common.BlockAndReceipts) error {
-	defer core.LogMethodDuration(rc.logger, measure.NewStopwatch(), "Rollup consumer processed block", log.BlockHashKey, b.BlockHeader.Hash())
+// ProcessBlobsInBlock - FIXME
+func (rc *rollupConsumerImpl) ProcessBlobsInBlock(ctx context.Context, b *common.BlockAndReceipts, blobs []*kzg4844.Blob) error {
+	defer core.LogMethodDuration(rc.logger, measure.NewStopwatch(), "Rollup consumer processed blobs", log.BlockHashKey, b.BlockHeader.Hash())
 
-	rollups := rc.extractRollups(b)
+	rollups, err := rc.extractAndVerifyRollups(b, blobs)
+	if err != nil {
+		rc.logger.Error("Failed to extract rollups from block", log.BlockHashKey, b.BlockHeader.Hash(), log.ErrKey, err)
+		return err
+	}
 	if len(rollups) == 0 {
+		rc.logger.Info("No rollups found in block", log.BlockHashKey, b.BlockHeader.Hash())
```

Comment:
Is this an "Info"?
I'd say it's quite normal for blocks not to have rollups. 
Or did we already expect there to be a rollup when we entered this method? But I don't think so?

I'd say it should be debug or trace